main.js

```diff
@@ -160,4 +160,146 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1a: Hoeveel exemplaren moeten we nog verkopen? Schrijf een functie die dit berekent.
+// aantal nog te verkopen units is gelijk aan (originalStock - sold) per type en dan totaal berekenen.
+const typesToSell = inventory.map((unit) => {
+  return unit.originalStock - unit.sold
+})
+console.log(typesToSell)
+
+const totalUnitsToSell = typesToSell.reduce((accumulator, currentValue) => {
+  return accumulator + currentValue;
+})
+console.log(totalUnitsToSell);
+
```

Comment:
Wat je hier in twee array-methods doet, kan ook in 1 functie uitgewerkt worden.
inventory mep, makt een array waarbij de resulataten van '`originalstock - sold`' worden bewaard, daarna worden die opgeteld in de reduce-method. In de reduce-methode kun je die berekeing in een keer doen. 

Om de data in de DOM te plaatsen gebruik je '`input1`' . Hier zou ik ook `totalUnitToSell`. Dan is het makkelijker in de html-code terug te vinden.